packages/material-ui/src/TableRow/TableRow.js

```diff
@@ -15,13 +16,13 @@ export const styles = theme => ({
     '&$selected': {
       backgroundColor:
         theme.palette.type === 'light'
-          ? 'rgba(0, 0, 0, 0.04)' // grey[100]
+          ? fade(theme.palette.primary.main, 0.04)
```

Comment:
Should it be primary or secondary? I believe it should match the color of the checkboxes. Should we support primary and secondary?

Reply:
According to [MDC](https://material-components.github.io/material-components-web-catalog/#/component/data-table), it should be primary (same color as checkbox). I don't think we need support for secondary in this case, people can always overwrite the default behavior.

Reply:
In this case, we have an issue with the demos:

![Capture d’écran 2020-01-20 à 13 50 58](https://user-images.githubusercontent.com/3165635/72727730-f11e2900-3b8b-11ea-9b86-afa6f18b6a5f.png)

Also, let's use http://material.io/ as a reference.

Reply:
Oh...sorry, I thought that the checked color from the checkbox was the primary but is the secondary, which means that secondary should be used. I've updated and also update hover css to be applied only to unselected rows.

Reply:
It's no obvious which color we should use. I personally start new projects with:
```jsx
  props: {
    MuiRadio: {
      color: 'primary',
    },
    MuiCheckbox: {
      color: 'primary',
    }
    MuiSwitch: {
      color: 'primary',
    }
  },
```

Reply:
indeed it's not obvious...but [this example](https://material.io/components/selection-controls/#theming) uses secondary. Is there a way to know which color the checkbox is using so that we use this color?

Reply:
What makes you conclude it's the secondary one?

> This travel app’s selection controls have been customized using Material Theming to use custom color.

Reply:
Right after that there is a "Color" section where it states 

> Crane’s selection controls use a custom color.

And there you can see that the custom color is 'Secondary'

Reply:
> And there you can see that the custom color is secondary.

I believe the secondary color can be blue and the primary red-orange. I doubt we can conclude.

> So, I think that we should use secondary color for selected state as it's the default color for selected state from Checkbox

Yes, definitely if we want Material-UI to be consistent. I'm wondering about what the specification would encourage 🤔.

Reply:
Ok, you were right, Crane secondary's color is red: https://material.io/design/material-studies/crane.html#color.

![Capture d’écran 2020-01-21 à 11 39 37](https://user-images.githubusercontent.com/3165635/72798168-c2af5500-3c42-11ea-8b36-8a8e010b6dd9.png)

Reply:
My point was this picture:
![image](https://user-images.githubusercontent.com/4071037/72798837-3736c380-3c44-11ea-9619-065a845e72f7.png)

What got my attention was not the color but the variant: `secondary` and from the yours it says: 

> which it uses to highlight selected items

So I think we are fine with secondary.

Reply:
Awesome, thanks for the precision.